release_notes_generator/builder.py

```diff
@@ -24,6 +24,7 @@
 from release_notes_generator.action_inputs import ActionInputs
 
 
+# pylint: disable=too-few-public-methods
```

Comment:
Doesn't have to be a class, since there is just one method with no params.

Reply:
Added TODO for future class reduction when following development does not provide reason why to keep as a class.